src/admin-views/tribe-options-addons-api.php

```diff
@@ -5,6 +5,26 @@
  */
 $internal = array();
 
+$current_url = add_query_arg(
+	array(
+		'page' => 'tribe-common',
+		'tab' => 'addons',
+		'post_type' => Tribe__Events__Main::POSTTYPE,
+	),
+	admin_url( 'edit.php' )
+);
+
+if (
+	isset( $_GET['action'] )
+	&& isset( $_GET['_wpnonce'] )
+	&& 'disconnect-facebook' === $_GET['action']
+	&& wp_verify_nonce( $_GET['_wpnonce'], 'disconnect-facebook' )
+) {
+	tribe_update_option( 'fb_token', '' );
+	wp_redirect( $current_url );
+	die;
+}
```

Comment:
I relocated some stuff to `Tribe__Events__Aggregator__Settings` - it seemed like the better place. As such, this is ready for another look-see.